### PR TITLE
refactor: remove litellm dependencies

### PR DESCRIPTION
**Pull request by nqngo**
Remove LiteLLM as a dependency for the Discord bot as it is now running in a separate docker container.

See:

https://github.com/bifrostlab/deploy-llm-bot/pull/12

### CODE REVIEW

**Review by samhwang — CHANGES_REQUESTED** [review]
Tests broken

**Comment by nqngo** [comment]
> Tests broken

Remove test, no problem. Jk let me fix it.

**Comment by rodonguyen** [comment]
I don't know much about Docker stuff but it looks good XD (as long as it works)

**Comment by nqngo** [comment]
Per last week discussion, the test is removed from github actions. Will need to rewrite the test to mock to ensure we can test functionality without calling the actual LiteLLM proxy.

File: tests/test_discord_bot_llm.py
Comment (samhwang) [inline tests/test_discord_bot_llm.py:1]: these changes aren't needed

File: Dockerfile.test_ollama
Comment (samhwang) [inline Dockerfile.test_ollama:1]: these changes aren't needed

**Review by samhwang — CHANGES_REQUESTED** [review]

**Review by samhwang — APPROVED** [review]

**Pull request merged**